### PR TITLE
feat: update blog post item

### PR DESCRIPTION
**Pull request by jinhojang6**
Preview:

Screenshot 2023-06-20 at 7 00 50 PM

Solves https://github.com/acid-info/logos-docusaurus-plugins/issues/69

### CODE REVIEW

**Comment by jinhojang6** [comment]
Updating the blog page

**Comment by jinhojang6** [comment]
Added a commit based on the new blog page design

Screenshot 2023-06-21 at 5 28 05 PM

<br/>

Todo: The header of the blog page
Screenshot 2023-06-21 at 5 28 44 PM

Issue:
1. Should it be handled by preset or let BU set it on their own?
2. If it should be handled by preset, what title and description should it use?

**Comment by amirhouieh** [comment]
> 
@jeangovil @jinhojang6 
This seems to be a bit too invasive approach. What if we just handle it by having a `/blog/index.mdx` and in there we use two components :
<BlogHeader/>
Array of <BlogItems/>

File: packages/logos-docusaurus-theme/src/client/theme/BlogPostItems/index.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { BlogPostProvider } from '@docusaurus/theme-common/internal'
+import BlogPostItem from '@theme/BlogPostItem'
+
+export default function BlogPostItems({
```
[the remark this answers is not in the document]
Comment (jinhojang6) [reply packages/logos-docusaurus-theme/src/client/theme/BlogPostItems/index.tsx:5]: I put the header in this component before and forgot to remove it after moving the header to bloglayout. Removed it

@jeangovil

File: packages/logos-docusaurus-theme/src/client/theme/BlogLayout/index.tsx [hunk 2]
```diff
@@ -1,40 +1,62 @@
...
-  )
-}
+import { Typography } from '@acid-info/lsd-react'
+import styles from './styles.module.scss'
+import Layout from '@theme/Layout'
+import BlogSidebar from '@theme/BlogSidebar'
+import { useBlogPageData } from './BlogPage.context'
 
 export default function BlogLayoutWrapper(props: Props): JSX.Element {
   const { sidebar, toc, children, ...layoutProps } = props
```
[the remark this answers is not in the document]
Comment (jinhojang6) [reply packages/logos-docusaurus-theme/src/client/theme/BlogLayout/index.tsx:12]: Updated

File: packages/logos-docusaurus-theme/src/client/theme/BlogPostItem/Header/index.tsx [hunk 3]
```diff
@@ -35,7 +49,10 @@ export default function BlogPostItemHeader(): JSX.Element {
         />
       )}
       <BlogPostItemHeaderTitle />
+      {frontMatter?.description && isBlogPostPage && _description}
       <BlogPostItemHeaderInfo />
+      {frontMatter?.description && !isBlogPostPage && _description}
```
[the remark this answers is not in the document]
Comment (jinhojang6) [reply packages/logos-docusaurus-theme/src/client/theme/BlogPostItem/Header/index.tsx:54]: updated

File: packages/logos-docusaurus-theme/src/client/theme/BlogPostItem/Header/Title/index.tsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+      {isBlogPostPage ? (
+        title
+      ) : (
+        <Link itemProp="url" to={permalink}>
+          {title}
+        </Link>
+      )}
+    </TitleHeading>
+  )
+}
```
[the remark this answers is not in the document]
Comment (jinhojang6) [reply packages/logos-docusaurus-theme/src/client/theme/BlogPostItem/Header/Title/index.tsx]: Updated

[the remark this answers is not in the document]
Comment (jinhojang6) [reply packages/logos-docusaurus-theme/src/client/theme/BlogPostItem/Header/Title/index.tsx]: update

**Pull request merged**